text/2018-10-9-reduce-one-copy-in-grpc-send.md

```diff
@@ -0,0 +1,120 @@
+# Summary
+
+Reduce one data copy in the process of sending message in grpcio.
+This is not a new feature but an optimization.
+
+This change is included in PR [#238][#238].
+Previous work: [#225][#225], [#235][#235].
+
+ [#225]: https://github.com/pingcap/grpc-rs/pull/225
+ [#235]: https://github.com/pingcap/grpc-rs/pull/235
+ [#238]: https://github.com/pingcap/grpc-rs/pull/238
+
+# Motivation
```

Comment:
Usually, use only one first-level heading within one document. For others, you can use second-level...

Reply:
This is what the template look like. @siddontang

Reply:
Got it.